 __tests__/pet.test.js

```diff
@@ -0,0 +1,247 @@
+const Pet = require("../src/pet");
+
+describe("constructor", () => {
+  it("returns an object", () => {
+    expect(new Pet("Ted")).toBeInstanceOf(Object);
+  });
+});
+
+describe("constructor", () => {
+  it("sets the name property", () => {
+    const pet = new Pet("Ted");
+
+    expect(pet.name).toEqual("Ted");
+  });
+});
+
+describe("constructor", () => {
+  it("has an initial age of 0", () => {
+    const pet = new Pet("Ted");
+
+    expect(pet.age).toEqual(0);
+  });
+});
+
+describe("constructor", () => {
+  it("has an initial hunger of 0", () => {
+    const pet = new Pet("Ted");
+
+    expect(pet.hunger).toEqual(0);
+  });
+});
+
+describe("constructor", () => {
+  it("has an initial fitness of 10", () => {
+    const pet = new Pet("Ted");
+
+    expect(pet.fitness).toEqual(10);
+  });
+});
```

Comment:
You dont need to create separate describe blocks for each test that belongs to the same group, you can put these all together like this:
```
describe("constructor", () => {
  it("returns an object", () => {
    expect(new Pet("Ted")).toBeInstanceOf(Object);
  });

  it("sets the name property", () => {
    const pet = new Pet("Ted");

    expect(pet.name).toEqual("Ted");
  });

  it("has an initial age of 0", () => {
    const pet = new Pet("Ted");

    expect(pet.age).toEqual(0);
  });

  it("has an initial hunger of 0", () => {
    const pet = new Pet("Ted");

    expect(pet.hunger).toEqual(0);
  });

  it("has an initial fitness of 10", () => {
    const pet = new Pet("Ted");

    expect(pet.fitness).toEqual(10);
  });
});
```